documentation/projects/proposals/ingestion_server_removal/20240328-implementation_plan_ingestion_server_removal.md

```diff
@@ -625,6 +609,13 @@ potential suggestions for that time:
   would involve expanding the indexer workers' API to handle this additional
   task.
 
+### ASG approach
+
+An earlier draft of this implementation plan used an AutoScaling Group for each
+environment. Airflow would use `set_desired_capacity` to set the ASG's capacity,
+and the ASG itself would then spin up (and down) EC2 instances. A disadvantage
+of this approach was that it is impossible to retry a single indexer worker.
+
```

Comment:
Nice, this looks good. I've been meaning to suggest we find a standardized way to denote changes to our IPs and other documents. Perhaps a "changelog" section that links to the PR file diff and summarizes the changes?

What you've done seems sufficient for now, but reminded me to create an issue for this, so thanks :smile: 

https://github.com/WordPress/openverse/issues/4619